File: src/IsbInterop/ComponentTokens/Wrappers/ComponentTokenInfo.vb
Imports IsbInterop.Base.Wrappers

Namespace ComponentTokens.Wrappers

  ''' <summary>
  ''' Обертка над IComponentTokenInfo.
  ''' </summary>
  Friend Class ComponentTokenInfo
    Inherits EdmsObjectInfo
    Implements IComponentTokenInfo

    ''' <summary>
    ''' Конструктор.
    ''' </summary>
    ''' <param name="rcwIComponentTokenInfo">COM-объект IComponentTokenInfo.</param>
    ''' <param name="scope">Область видимости.</param>
    Public Sub New(rcwIComponentTokenInfo As Object, scope As IScope)
      MyBase.New(rcwIComponentTokenInfo, scope)
    End Sub
  End Class
End NameSpace

File: src/IsbInterop.CSharp/ComponentTokens/Wrappers/ComponentTokenInfo.cs
using IsbInterop.Base.Wrappers;

namespace IsbInterop.ComponentTokens.Wrappers
{
  /// <summary>
  /// Обертка над IComponentTokenInfo.
  /// </summary>
  internal class ComponentTokenInfo : EdmsObjectInfo, IComponentTokenInfo
  {
    /// <summary>
    /// Конструктор.
    /// </summary>
    /// <param name="rcwIComponentTokenInfo">COM-объект IComponentTokenInfo.</param>
    /// <param name="scope">Область видимости.</param>
    public ComponentTokenInfo(object rcwIComponentTokenInfo, IScope scope)
      : base(rcwIComponentTokenInfo, scope) { }
  }
}
